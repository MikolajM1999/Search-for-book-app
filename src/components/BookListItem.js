import React from 'react'

const BookListItem = ({ bookImageLink, bookFullTitle, bookShortDescription }) => (
   <div>
      <img src={bookImageLink} alt={bookFullTitle} width="128" height="194" />
      <h2>{bookFullTitle}</h2>
      <p>{bookShortDescription}</p>
   </div>
)

export default BookListItem