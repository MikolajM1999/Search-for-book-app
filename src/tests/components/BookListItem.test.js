import React from 'react'
import { shallow } from 'enzyme'
import BookListItem from '../../components/BookListItem'

test('BookListItem renders without crashing', () => {
   shallow(<BookListItem />)
})

test('should render BookListItem correctly', () => {
   const wrapper = shallow(<BookListItem />)
   expect(wrapper).toMatchSnapshot()
})