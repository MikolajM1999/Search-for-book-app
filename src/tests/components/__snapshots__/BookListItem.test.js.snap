// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render BookListItem correctly 1`] = `
<div
  className="book container-for-books__book-item"
>
  <a
    className="book__image-link"
    rel="noopener noreferrer search"
    target="_blank"
  >
    <img
      height="194"
      width="128"
    />
  </a>
  <div
    className="div-info book__right-side-info"
  >
    <p
      className="div-info__item book__published-date"
    />
    <h3
      className="div-info__item book__full-title"
    />
    <p
      className="div-info__item book__short-description"
    />
  </div>
</div>
`;
